build/optaplanner-build-parent/pom.xml

```diff
@@ -1131,49 +951,6 @@
         </plugins>
       </build>
     </profile>
-    <profile>
-      <id>generate-revapi-report</id>
-      <!-- There is no need to run tests when generating the report -->
-      <properties>
-        <skipTests>true</skipTests>
-      </properties>
```

Comment:
I suspect revapi will best stay (see previous comment)

Reply:
Yes, this profile is completely unrelated to the regular revapi check we do both locally and in the PR checks.

We don't use the reporting capabilities of revapi anymore. In the past we did use it to generate a list of changes between two product releases. In the OptaPlanner, we have something much better - the upgrade-recipe.